server/src/main/java/org/spine3/server/validate/EnumFieldValidator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.validate;
+
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Descriptors;
+import org.spine3.base.FieldPath;
+import org.spine3.validate.ConstraintViolation;
+
+import java.util.List;
+
+/**
+ * Validates fields of type {@link Descriptors.EnumValueDescriptor}.
+ *
+ * @author Dmitry Kashcheiev
+ */
+/* package */ class EnumFieldValidator extends FieldValidator<Descriptors.EnumValueDescriptor>  {
+
+    /**
+     * Creates a new validator instance.
+     *
+     * @param descriptor a descriptor of the field to validate
+     * @param fieldValues values to validate
+     * @param rootFieldPath a path to the root field (if present)
+     */
+    /*package*/ EnumFieldValidator(Descriptors.FieldDescriptor descriptor, ImmutableList<Descriptors.EnumValueDescriptor> fieldValues, FieldPath rootFieldPath) {
+        super(descriptor, fieldValues, rootFieldPath, false);
+    }
+
+    @Override
+    protected boolean isValueNotSet(Descriptors.EnumValueDescriptor value) {
+        return value.getNumber() == 0;
```

Comment:
Have `final boolean result` field immediately before return. This way it's easier to debug, when needed.